Documentation/deploy-master.md

```diff
@@ -88,11 +89,14 @@ The [kubelet](http://kubernetes.io/docs/admin/kubelet.html) is the agent on each
 
 On the master node, the kubelet is configured to communicate with the API server, but not register for cluster work, as shown in the `--register-schedulable=false` line in the YAML excerpt below. This prevents user pods being scheduled on the master nodes, and ensures cluster work is routed only to task-specific worker nodes.
 
+When using Calico, the kubelet is configured to use CNI for networking. This makes Calico aware of each pod that is created and allows it to network the pods into the flannel overlay. The Calico CNI plugin is actually called by the Flannel CNI plugin so that the correct IP range that flannel has selected for the host can be used.
+
 Note that the kubelet running on a master node may log repeated attempts to post its status to the API server. These warnings are expected behavior and can be ignored. Future Kubernetes releases plan to [handle this common deployment consideration more gracefully](https://github.com/kubernetes/kubernetes/issues/14140#issuecomment-142126864).
 
 * Replace `${ADVERTISE_IP}` with this node's publicly routable IP.
 * Replace `${DNS_SERVICE_IP}`
-* Replace `${K8S_VER}` This will map to: `quay.io/coreos/hyperkube:${K8S_VER}` release
+* Replace `${K8S_VER}` This will map to: `quay.io/coreos/hyperkube:${K8S_VER}` release. If using Calico a version that includes CNI binaries should be used. e.g. `v1.2.2_coreos_cni.0`
```

Comment:
Is there a way we can always build in the CNI binaries? And then enable them for Calico users? Or make Calico the default? This choice is going to be confusing for users.